code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm

```diff
@@ -12,7 +12,7 @@
 	ui_auto_update = FALSE
 
 	var/page = "main" //main - Main Menu, order - Order Page, item_details - Item Details Page, tracking - Tracking Page
-	var/selected_category = "" // Category that is currently selected
+	var/selected_category = "security" // Category that is currently selected
```

Comment:
Should probably pick one category from the repository, instead of being hardcoded (as a string), or you can also do it based on the typepath initial value